### DISCUSSION TITLE
install package

### DISCUSSION THREAD

**Discussion by kotovasia5120** [bol-van/zapret#227] [Q&A] [answered]
Hello, I have problems installing packages, am I doing something wrong?

```shell
~/git/zapret # ./blockcheck.sh
* checking system
Linux detected
firewall type is nftables
* checking privileges
* checking prerequisites
nftables queue support is not available. pls install modules.

press enter to continue

~/git/zapret # ./install_prereq.sh
* checking system
system is not either systemd, openrc or openwrt based
easy installer can set up config settings but can't configure auto start
you have to do it manually. check readme.txt for manual setup info.
system is based on linux
* checking privileges

select firewall type :
1 : iptables
2 : nftables
your choice (default : nftables) : 1
selected : iptables
* checking prerequisites
required utilities exist : curl ip6tables ipset iptables

press enter to continue

~/git/zapret # opkg install curl  nft
Package curl (8.6.0-1) installed in root is up to date.
Unknown package 'nft'.
Collected errors:
 * opkg_install_cmd: Cannot install package nft.
~/git/zapret # opkg install curl ip6tables ipset iptables
Package curl (8.6.0-1) installed in root is up to date.
Unknown package 'ip6tables'.
Package ipset (7.17-1) installed in root is up to date.
Package iptables (1.4.21-4) installed in root is up to date.
Collected errors:
 * opkg_install_cmd: Cannot install package ip6tables.
~/git/zapret #
```  

![image](https://github.com/user-attachments/assets/e38f262c-f582-4d83-b411-63a864a02077)

**Comment by bol-van** [comment]
if you run zapret on an unsupported system
such as entware on keenetic
you must know how to install modules
i dont know

**Reply by bol-van** [reply]
kernel modules is not something you can install on entware
they are bound to kernel

**Reply by kotovasia5120** [reply]
@bol-van  As far as I understand, the module is installed but is not visible to the script. Is this critical?
When calling the ip6tables command, it asks to specify arguments.

**Comment by bol-van** [comment]
you mix nf and ip tables. choose one.
what is your system ?

**Reply by kotovasia5120** [reply, marked as answer]
I just tried every option.
After deleting the repository and the uninstall script, the request to install the package disappeared.
Keenetic Giga (KN-1011)

**Reply by Juster737** [reply]
> I just tried every option. After deleting the repository and the uninstall script, the request to install the package disappeared. Keenetic Giga (KN-1011)

Man, what you delete? How?

**Reply by kotovasia5120** [reply]
> > I just tried every option. After deleting the repository and the uninstall script, the request to install the package disappeared. Keenetic Giga (KN-1011)
> 
> Man, what you delete? How?

uninstall_easy.sh.

and delite

opt/zapret
git/zapret
